### PR TITLE
marshal os version to json

### PR DESCRIPTION
**Pull request by joesis**

### CODE REVIEW

**Comment by joesis** [comment]
If time allows, let me see if I can add some test.

File: check/check_test.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestToJson(t *testing.T) {
+	var p Params
+	assert.NoError(t, p.fill(""), "should have no error filling params")
+	body, err := json.Marshal(p)
+	assert.NoError(t, err, "should have no error marshalling params to JSON")
+	_ = body
```
Comment (oxtoacart) [inline check/check_test.go]: I don't understand why this is needed?

Comment (joesis) [reply check/check_test.go]: Ha I forgot to remove this

**Comment by joesis** [comment]
Removed the line. Merging. Thanks @oxtoacart !

**Pull request merged**